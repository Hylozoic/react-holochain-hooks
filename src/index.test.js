import 'react'
import useHolochainWebClient from './index'
import { shallow } from 'enzyme';
import React from 'react'

function TestComponent () {
  const { callZomeRef, callRef, closeRef } = useHolochainWebClient('ds;akdls')
  callZomeRef.current()
  callRef.current()
  closeRef.current()
  return <div />
}

it('imports the file', () => {  
  const wrapper = shallow(<TestComponent />)
  expect(wrapper).toMatchSnapshot()
})