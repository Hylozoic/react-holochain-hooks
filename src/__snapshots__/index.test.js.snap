// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`imports the file 1`] = `ShallowWrapper {}`;
